### PR TITLE
Deletion of domain tooltip strings, addition of new modal design strings

### PR DESCRIPTION
**Pull request by rafeerahman**
MPP-3239 - This task required removal of a tool-tip that appears when clicking your relay domain name. I deleted the strings corresponding to this tool-tip in misc.ftl 

MPP-3261 - Updated the strings in modals.ftl to match the new design for generating a mask.

### CODE REVIEW

**Review by peiying2 — CHANGES_REQUESTED** [review]
I called out a few things.  The main issue is reusing the same string ID with different string content. This is a big no no. You can use the same string ID with a version indication such as `-v2` as a suggestion. 

Here is a doc for developer using Fluent: https://github.com/mozilla-l10n/documentation/blob/main/src/localization/dev_best_practices.md#fluent-files.

**Comment by rafeerahman** [comment]
> I called out a few things. The main issue is reusing the same string ID with different string content. This is a big no no. You can use the same string ID with a version indication such as `-v2` as a suggestion.
> 
> Here is a doc for developer using Fluent: https://github.com/mozilla-l10n/documentation/blob/main/src/localization/dev_best_practices.md#fluent-files.

Thank for the documentation and checking this. For the deletions I made, do you recommend marking them with `# DEPRECATED` instead of deleting them?

**Comment by rafeerahman** [comment]
Not ready for merge yet, making some changes

**Comment by peiying2** [comment]
> Thanks for the documentation and checking this. For the deletions I made, do you recommend marking them with `# DEPRECATED` instead of deleting them?

- If you label a deprecated string instead of removing it, the old version will remain until the new version is localized, which will replace the deprecated string. Choosing this approach means it requires regular cleanup so old strings won't be localized for communities who are just beginning to localize the product. 
- If you remove it, the English string will show up in the localized page (mix of English and localized text). This happens all the time even without the new strings replacing the old. Every time new strings are added to an existing page, it takes a while for the communities to catch up.

In case you wonder, the content of localized strings that are removed from the product will remain in Pontoon as translation memory (TM), so when similar or the same strings appear for any products, not just Relay, they can be leveraged or referenced for consistency.

File: en/modals.ftl [hunk 1]
```diff
@@ -4,13 +4,15 @@
 
 ## Modal for generating a custom mask
 
+modal-custom-alias-picker-tip = Tip: To create a custom mask anytime, make it up on the spot. If you use your Relay domain, it’ll work, even if you didn’t generate it here first.
```
Comment (peiying2) [inline en/modals.ftl:7]: Relay, the brand, must follow Fluent branding format.

File: en/modals.ftl [hunk 2]
```diff
@@ -4,13 +4,15 @@
...
+
 modal-custom-alias-picker-heading-2 = Create a new custom mask
 modal-custom-alias-picker-warning-2 = All you need to do is make up and share a unique mask that uses your custom subdomain — the mask will be generated automatically. Try “shop@customsubdomain.mozmail.com” next time you shop online, for example.
 modal-custom-alias-picker-form-heading-2 = Or, create a custom mask manually
-modal-custom-alias-picker-form-prefix-label-2 = Enter email mask prefix
+modal-custom-alias-picker-form-prefix-label-v3 = Enter the text that goes before the @ symbol:
```
Comment (peiying2) [inline en/modals.ftl:12]: You may need to check with other devs on the convention to indicate version of a string. Looks like `-3` might be the way Relay team handles it so it stays consistent with the strings around this one.

**Review by peiying2 — CHANGES_REQUESTED** [review]
Mozilla and Firefox product brand names need to be coded in Fluent format.

**Comment by rafeerahman** [comment]
I have added the requested changes, thank you for reviewing

File: en/modals.ftl [hunk 3]
```diff
@@ -4,13 +4,15 @@
 
 ## Modal for generating a custom mask
 
+modal-custom-alias-picker-tip = Tip: To create a custom mask anytime, make it up on the spot. If you use your { -brand-name-firefox-relay } domain, it’ll work, even if you didn’t generate it here first.
```
Comment (peiying2) [inline en/modals.ftl:7]: @rafeerahman I did a check of the reference to domain, it usually use Relay, not Firefox Relay: https://pontoon.mozilla.org/zh-CN/firefox-relay-website/all-resources/?search=domain&string=243297 - see the search result on the left panel. We probably should change to `{ -brand-name-relay } domain`. The previous comment was an example what Fluent brand format looks like.

**Review by peiying2 — APPROVED** [review]
Thank you making all the changes. It will get easier!

File: en/modals.ftl (hunk 3 above)
Comment (rafeerahman) [reply en/modals.ftl]: I see, `{ -brand-name-relay }` is correct. Is it ok to use https://github.com/mozilla-l10n/fx-private-relay-l10n/commit/1ebc8676877ff061e2ef2e5c615332dee8b4893b as a reference for the future?

Comment (peiying2) [reply en/modals.ftl]: Please refer to this file: https://github.com/mozilla-l10n/fx-private-relay-l10n/blob/main/en/brands.ftl - the section in the file you mentioned should have been removed after the brands.ftl was created. This way, we can stay consistent across all the files to refer to this one only. One change will affect all the others.

**Pull request merged**